geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearDUnitTest.java

```diff
@@ -211,8 +357,52 @@ public void afterRegionClear(RegionEvent event) {
       clears.incrementAndGet();
     }
 
+    @Override
+    public void afterRegionDestroy(RegionEvent event) {
+      Region region = event.getRegion();
+      logger.info("Region " + region.getFullPath() + " is destroyed.");
+      destroyes.incrementAndGet();
+    }
+
     int getClears() {
       return clears.get();
     }
+
+    int getDestroys() {
+      return destroyes.get();
+    }
+  }
+
+  public static HashMap<String, AtomicInteger> clearsByRegion = new HashMap<>();
+  public static HashMap<String, AtomicInteger> destroyesByRegion = new HashMap<>();
```

Comment:
Typo here, this should be `destroysByRegion`.

Reply:
I will fix it.